v20.2/backup.md

```diff
@@ -35,9 +35,14 @@ To view the contents of an enterprise backup created with the `BACKUP` statement
 
 ## Required privileges
 
-- Only members of the `admin` role can run `BACKUP`. By default, the `root` user belongs to the `admin` role.
+- Full cluster backups can only be run by members of the `admin` role. By default, the `root` user belongs to the `admin` role.
+- For all other backups, the user must have read access (`SELECT` or `USAGE`) on all objects being backed up.
```

Comment:
~By "all other" do we only mean "incremental"?  If so, would it be clearer to say "for incremental backups"?  If so that could be a link to #backup-types or #incremental-backups